packages/web/src/lib/web-mode.ts

```diff
@@ -269,4 +269,46 @@ export class WebMode {
 		}
 		throw new Error("Transaction Error");
 	}
+
+	async makeTx(transactions: ExecParams[]): Promise<Transaction[]> {
+		const txns: Transaction[] = [];
+		for (const [_, txn] of transactions.entries()) {
+			txns.push(mkTransaction(txn, await this.getSuggestedParams(txn.payFlags)));
+		}
+		return txns;
+	}
+	async signTx(transaction: algosdk.Transaction): Promise<SignedTransaction> {
+		const binaryTx = transaction.toByte();
+		const base64Tx = this.algoSigner.encoding.msgpackToBase64(binaryTx);
+		const signedTx = await this.algoSigner.signTxn([
+			{
+				txn: base64Tx,
+			},
+		]);
+		const signedTxJson = signedTx as JsonPayload;
```

Comment:
why do we need to use `as` here? Normally we should avoid using `as` whenever possible.